Remotion/Web/Development.WebTesting/WebDriver/Configuration/Chrome/ChromiumUserDirectoryCleanUp.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using JetBrains.Annotations;
+using log4net;
+using Remotion.Utilities;
+
+namespace Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chrome
+{
+  public class ChromiumUserDirectoryCleanUp : IBrowserSessionCleanUpStrategy
```

Comment:
Missing Documentation?

Reply:
Fixed in https://github.com/re-motion/Trunk-SVN2Git/pull/9/commits/8ba221047ebbc3b720b6953cb20ee8fcb115e990.